pkgs/development/tools/selenium/server/default.nix

```diff
@@ -25,9 +27,9 @@ in stdenv.mkDerivation rec {
     mkdir -p $out/share/lib/${name}
     cp $src $out/share/lib/${name}/${name}.jar
     makeWrapper ${jre}/bin/java $out/bin/selenium-server \
-      --add-flags "-cp ${htmlunit-driver}/share/lib/${htmlunit-driver.name}/${htmlunit-driver.name}.jar:$out/share/lib/${name}/${name}.jar" \
+      --add-flags "-cp $out/share/lib/${name}/${name}.jar:${htmlunit-driver}/share/lib/${htmlunit-driver.name}/${htmlunit-driver.name}.jar" \
       --add-flags ${optionalString chromeSupport "-Dwebdriver.chrome.driver=${chromedriver}/bin/chromedriver"} \
-      --add-flags "org.openqa.grid.selenium.GridLauncher"
+      --add-flags "org.openqa.grid.selenium.GridLauncherV3"
```

Comment:
it looks like `pkgs/development/tools/selenium/selendroid/default.nix` needs also the new `GridLauncherV3` argument.

Reply:
Thx for the hint. I updated it as well.